packages/gestalt/src/Link/Link.js

```diff
@@ -53,8 +54,21 @@ export default class Link extends React.Component<Props, State> {
   };
 
   render() {
-    const { children, inline = false, target = null, href } = this.props;
-    const rel = target === 'blank' ? 'noopener noreferrer' : null;
+    const {
+      children,
+      inline = false,
```

Comment:
Note to self - we should be using `defaultProps` instead of destructuring the props and setting default values.

https://stackoverflow.com/a/42733457/117193

Reply:
That link is pretty... light on any details. Claims about perf without any numbers aren't particularly convincing. Claims about correctness without pointing out any actual bugs aren't convincing either :/